aws/resource_aws_storagegateway_gateway_test.go

```diff
@@ -1061,10 +1110,10 @@ resource "aws_storagegateway_gateway" "test" {
 `, rName)
 }
 
-func testAccAWSStorageGatewayGatewayConfig_DirectoryServiceSimpleDirectory(rName, domain string) string {
+func testAccAWSStorageGatewayGatewayConfig_DirectoryServiceSimpleDirectory(rName, domainName string) string {
 	return fmt.Sprintf(`
 resource "aws_directory_service_directory" "test" {
-  name     = %[2]q
+  name     = "terraformtesting.com"
```

Comment:
This should use the `domainName` parameter

```suggestion
  name     = %[2]q
```

Reply:
Confirmed, the AccTests caught that. Fixed.